target/linux/ramips/image/mt7621.mk

```diff
@@ -378,6 +378,19 @@ define Device/beeline_smartbox-turbo
 endef
 TARGET_DEVICES += beeline_smartbox-turbo
 
+define Device/beeline_smartbox-turbo-plus
+  $(Device/sercomm_cxx)
+  IMAGE_SIZE := 32768k
+  SERCOMM_HWID := CQR
+  SERCOMM_HWVER := 10000
+  SERCOMM_SWVER := 2010
+  DEVICE_VENDOR := Beeline
+  DEVICE_MODEL := SmartBox TURBO+
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7615-firmware \
```

Comment:
`kmod-mt7615e` can be removed now. Reference: https://github.com/openwrt/openwrt/commit/9a07895729a9701a673951a31ceb71f16f0a6fad

Reply:
Fix